backend/core/client.ts

```diff
@@ -610,6 +613,7 @@ export enum EnumAttachmentType {
   'ApplicationDownload' = 'ApplicationDownload',
   'ExternalApplication' = 'ExternalApplication'
 }
+export type CombinedUserTypes = (User & any) | null;
```

Comment:
This feels a little hacky to me, but I'm fine if we clean it up later.

Reply:
That's not me. It'ss autogenerated.